doc/faq.rst

```diff
@@ -224,7 +224,7 @@ In this case you can update it to look like this:
     (dirs :standard .cargo \ target)
     (data_only_dirs vendor)
 
-How Can I Write Inline Tests in a Package Without my Users Needing to Install ``ppx_inline_test``?
+How can I write inline tests in a package without my users needing to install ``ppx_inline_test``?
```

Comment:
Because there is apparently no answer to "how", I think we can just answer the why not?

```suggestion
Why can't I write inline tests in a package without users needing to install ``ppx_inline_test``?
```